### DISCUSSION TITLE
Times New Roman and 1,5 spacing

### DISCUSSION THREAD

**Discussion by rombasilio** [joaomlourenco/novathesis#372] [Q&A] [answered]
Dear friends, how to change its font to _Times_ and 1,5 spacing (nova-fcsh) Version 2022-10-06 [6.10.8]? Thank you!

**Comment by joaomlourenco** [comment, marked as answer]
@rombasilio 
The line spacing is by default 1.5.
To use the times font, open `Config/1_novathesis.tex`, and near line 109, uncomment and be sure it contains:
```latex
\ntsetup{fontstyle=newtx} 
```

**Reply by rombasilio** [reply]
Thank you so much! :)

**Reply by NoronhaC** [reply]
Hi Joao, 

Is it possible to change the font to Arial? 

Thank you
Pedro

**Reply by joaomlourenco** [reply]
Yes.  Uncomment and set `\ntsetup{style/font=arial}` near line 120 of `Config/1_novathesis.tex`.

**Reply by NoronhaC** [reply]
\ntsetup{fontstyle=newpx}   % The font set to be used in the document
%                               %     [ bookman, erewho, kpfonts, libertine,
%                               %       schola, stix2, newpx, newtx
%                               %       newsgot(*), calibr(*), kieranhealy(*) ] 
%                               % (*) available only for xelatex/lualatex)
%                               % DEFAULT: fontstyle=newpx

This? I changed "newpx" to arial and it does nothing

**Reply by joaomlourenco** [reply]
Well… let me complete my previous reply.
You can use arial, but you must have the font installed in your computer and accessible to LaTeX.  Then, you have to use `xelatex`or `lualatex` **and not** `padlatex`.

If this sounds too confusing for you, I suggest you *don't* use arial, and use one of the standard LaTeX fonts.

**Comment by NoronhaC** [comment]
Sorry, too confusing for me, my bad. Is it possible in overleaf version?

**Reply by NoronhaC** [reply]
Sorry how can I add arial in overleaf template? 
Regards

**Reply by NoronhaC** [reply]
Is it possible in overleaf version the arial font? Thank you

**Comment by figuenu** [comment]
Good morning,
Back to the Arial issue.
I am using Overleaf and according to the Nova/ENSP guidelines, the font to be used is Arial.
I tried to add \ntsetup{style/font=arial}  near line 120 of Config/1_novathesis.tex. However, it doesn't work and shows a first page with 
![image](https://github.com/user-attachments/assets/6447334f-10e3-4873-92a6-11556c5b83fa)

Can you please help?

[orientacoes-para-a-elaboracao-de-trabalhos-academicos.pdf](https://github.com/user-attachments/files/19159400/orientacoes-para-a-elaboracao-de-trabalhos-academicos.pdf)

Cumprimentos,
Nuno

**Reply by joaomlourenco** [reply]
To use the Arial font, [follow the instructions in the wiki](https://github.com/joaomlourenco/novathesis/wiki/using-the-arial-font).

**Reply by figuenu** [reply]
Dear João. Thank you very much
Regards
